js/components/findbar.js

```diff
@@ -60,6 +63,18 @@ class FindBar extends ImmutableComponent {
     input.select()
   }
 
+  componentWillMount () {
+    let stylesheet = document.getElementById('findBarStylesheet')
```

Comment:
Honestly I'd probably get around this problem just by changing the hover state, it's probably not worth having this to spec for the amount of time it can take to do it properly.

However if you want to, I think the proper way is to handle mouseEnter and moueLeave and set state in the `findDetail` for if it is hovered or not.  Then render based on that state.